### ISSUE TITLE
Backspace in insert mode not working

### ISSUE THREAD

**Issue by Oscarteg** [VundleVim/Vundle.vim#770]
After installing vundle my backspace isn't working in insert mode anymore. Removed the .vimrc and .vim directory and it's working again. Installing vundle again the same problem.

**Comment by AlexisFinn** [comment]
Try :verbose imap <BS> to see if a plugin is remapping it, end the try removing said plugin seing if that helps.
Repeat until this process until you can clearly define the culprit (wich may not be Vundle)

**Comment by jep-dev** [comment]
My Vundle does not add this mapping and appears not to provide any mappings at all (`:verbose map` shows my .vimrc, a few entries from /usr/share, and the rest from other bundles.) Please, if you can track down the cuplrit, mention them here and maybe shoot them a message - imapping backspace is unreasonable, IMHO - and if you can't, maybe we can troubleshoot from your .vimrc.

**Comment by Greatlemer** [comment]
Not sure if this will help but I was experiencing this same issue even without using vundle after I upgraded to vim 8.
I looked at things (using `:scriptnames`) and it seems that with vim 7 it was always loading /usr/share/vim/vimrc before ~/.vimrc which included the line:
`set backspace=2`
When I switched to vim 8 it no longer seemed to be loading a default file and ~/.vimrc is first.
However upon removing ~/.vimrc completely it then started loading /usr/local/share/vim/vim80/defaults.vim which once again includes a line:
`set backspace=indent,eol,start`
This is equivalent to `set backspace=2`.
I then explicitly added `set backspace=indent,eol,start` to my .vimrc and it now works fine so I'd suggest trying that if you don't already have an equivalent in your vimrc.
